substra/__version__.py

```diff
@@ -1 +1 @@
-__version__ = "0.53.0"
+__version__ = "0.54.0a1"
```

Comment:
I would not bet my life on this one, but as discussed I think that the version update is not mandatory as it will be installed from main on substrafl
```suggestion
__version__ = "0.53.0"
```